zedc/README.md

```diff
@@ -0,0 +1,32 @@
+# Zowe Explorer Development CLI
+
+A lightweight CLI that can be used to facilitate Zowe Explorer development and testing.
+
+## Requirements
+
+- `cmake, gcc`
+- Git Bash
+- Rust
+- Windows, macOS or Linux
+
+## Installation
+
+- Build using Cargo: `cargo build --release`
+- Copy the `zedc` binary from the `target/release` folder into its own folder
+- Add the folder for `zedc` to your `PATH` environment variable.
+
+## Command overview
+
+To view all available commands and arguments, run the command `zedc <command_name>` to view the built-in help. You can also run `zedc` to display all available top-level commands.
+
+- `setup`
+  - checkout a Git ref (optional)
+  - clean `node_modules`
+  - install the dependencies using the correct package manager
+- `pm`
+  - forwards commands to the package manager for the current branch
+- `test` - test a local VSIX or an artifact from GitHub using a Git ref
```

Comment:
Noticed a few quirks when running the `test` command:
* `zedc t ghr feat/zedc` - This fails to fetch artifacts, which may be expected because the latest commit on the branch didn't trigger a Zowe Explorer CI build?
* `zedc t ghr fix/extender-registration-failure` - This works since the latest commit on the branch has an associated workflow with a VSIX artifact. But I when I run the same command a 2nd time, I get the following error (not sure if it's a problem with the VS Code archive or the VSIX):
    ```
    📤 Unpacking VS Code archive...
    Error: File exists (os error 17)
    ```

Reply:
* The first command's behavior is intended by the current impl - as discussed offline this can be enhanced in the future to find the first commit in the branch that has a valid artifact 🙂 
* Thanks for giving context for the error - I've added some logic that will remove the dir. for the latest VS Code version if it already exists - addressed in 4c0b248